### PR TITLE
Bug 1887361: Move manifest to 4.7

### LINKED ISSUE #742: Please prepare bundle for OCP 4.7
Now that we are fully branched for 4.7, please prepare your operator to supply a 4.7 bundle, so that 4.7 operator publishing works and doesn't overwrite 4.6 bundles. This means at least updating the package.yaml under https://github.com/openshift/cluster-logging-operator/tree/master/manifests
@jcantrill @richm

Reference: https://github.com/openshift/ocp-build-data/pull/708

### PR DESCRIPTION
**Pull request by jcantrill**
This PR bumps the manifest to 4.7 and replaces https://github.com/openshift/cluster-logging-operator/pull/736

ref: https://bugzilla.redhat.com/show_bug.cgi?id=1887361

Fixes #742

### CODE REVIEW

File: manifests/cluster-logging.package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/clusterlogging.v4.6.0.clusterserviceversion.yaml
+#! package-manifest: ./deploy/chart/catalog_resources/rh-operators/clusterlogging.v4.7.0.clusterserviceversion.yaml
 packageName: cluster-logging
 channels:
 - name: "4.6"
```
Comment (periklis) [inline manifests/cluster-logging.package.yaml:4]: Shouldn't the channel name be also `4.7`?

File: Dockerfile.e2e [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # with the cluster logging operator source so we can use all the
 # test scripts to deploy the elasticsearch operator and then
 # test the clusterlogging operator
-FROM registry.svc.ci.openshift.org/ocp/4.6:elasticsearch-operator-src
+FROM registry.svc.ci.openshift.org/ocp/4.7:elasticsearch-operator-src
```
Comment (periklis) [inline Dockerfile.e2e:5]: FYI, this is actually the blocker that requires us to merge this manually.

**Review by periklis — CHANGES_REQUESTED** [review]

File: Dockerfile.e2e (hunk 2 above)
Comment (jcantrill) [reply Dockerfile.e2e]: reverted to 4.6 for now... should matter for CLO

**Comment by periklis** [comment]
Why does this PR delete:

**Comment by jcantrill** [comment]
> Why does this PR delete:
> image

@vimalk78 do you know why?  i just rebased and fixed up your PR

**Comment by vimalk78** [comment]
> Why does this PR delete:
> image



> > Why does this PR delete:
> > image
> 
> @vimalk78 do you know why? i just rebased and fixed up your PR

these files are not deleted, the bundle is generated again after changing the manifest folder in root of repo.

**Review by periklis — APPROVED** [review]
/lgtm

**Comment by ewolinetz** [comment]
/hold
please adjust the skip range to be 4.5 -> 4.7

**Pull request merged**